pkg/util/resource.go

```diff
@@ -265,6 +300,11 @@ func (r *Resource) AddPodRequest(podSpec *corev1.PodSpec) *Resource {
 	for _, container := range podSpec.InitContainers {
 		r.SetMaxResource(container.Resources.Requests)
 	}
+	// If Overhead is being utilized, add to the total requests for the pod.
+	// We assume the EnablePodOverhead feature gate of member cluster is set (it is on by default since 1.18).
+	if podSpec.Overhead != nil {
+		r.Add(podSpec.Overhead)
+	}
```

Comment:
The field `.spec.overhead` was introduced by K/K from v1.16. 

Are there risks if a cluster(<v1.16) joined Karmada?

Reply:
I think it's ok. It may lead some resource difference larger than before. The estimator will give a more strict result of available replicas. However, actually the estimator itself is not 100% accurate.